server/exports.lua

```diff
@@ -126,7 +126,7 @@ local function AddGangs(gangs)
             break
         end
 
-        if QBCore.Shared.Gangs[key] ~= nil then
+        if QBCore.Shared.Gangs[key] then
             message = "gang_exists"
             shouldContinue = false
             errorItem = items[key]
```

Comment:
```suggestion
            errorItem = gangs[key]
```

Line 125 should also be changed to this.